internal/extramanifest/extramanifest.go

```diff
@@ -179,79 +192,149 @@ func ApplyExtraManifest(ctx context.Context, dc dynamic.Interface, restMapper me
 	return nil
 }
 
-func (h *EMHandler) ValidateExtraManifestConfigmaps(ctx context.Context, content []ibuv1.ConfigMapRef, ibu *ibuv1.ImageBasedUpgrade) error {
+// ValidateExtraManifestConfigmaps validates IBU extramanifest configmaps and returns validation warning and error.
+// It iterates over all manifests included in the configmaps and perform Kubernetes Dry-run.
+// Errors from Dry-run apply are considered warnings. This also includes cases where CRDs are missing from the current
+// stateroot and dependent namespace does not exist on the current stateroot but is also not found in the configmaps.
+// Other validation failures are treated as errors, such as random chars, missing resource Kind, resource ApiVersion,
+// resource name or the presence of disallowed resource types
+func (h *EMHandler) ValidateExtraManifestConfigmaps(ctx context.Context, content []ibuv1.ConfigMapRef) (string, error) {
+	var errs = []string{}     // validation errors
+	var warnings = []string{} // validation warnings
```

Comment:
```suggestion
	var errs []string     // validation errors
	var warnings []string // validation warnings
```
safer empty slice declaration (according to [styleguide](https://google.github.io/styleguide/go/decisions#nil-slices))